homework-no-git/06_14_2018-homework/js/app.js

```diff
@@ -0,0 +1,9 @@
+// The difference between a parameter and an argument is that parameter is a discription of what needs to be there and and argument is what is actually being calculated.
+//a console.log you are unable to return and use the result
+const sumDigits = (num) => {
+	const number = num.toString()
+	const add = number.split()
+	console.log(add.reduce(add))
```

Comment:
did you try to run this code?  it generates an error.  delete your line 6 and replace it with a for loop.